pagantis/update.xml

```diff
@@ -85,17 +101,17 @@
      <folder>vmpayment</folder>
      <version>1.1.2</version>
      <downloads>
-         <downloadurl type="full" format="zip">http://demoshop.pagamastarde.com/joomla/plg_pagamastarde_v1.1.2.zip</downloadurl>
+         <downloadurl type="full" format="zip">http://demoshop.pagantis.com/joomla/plg_pagantis_v1.1.2.zip</downloadurl>
      </downloads>
      <tags>
          <tag>stable</tag>
      </tags>
      <maintainer>Albert Fatsini</maintainer>
-     <maintainerurl>https://www.pagamastarde.com/</maintainerurl>
+     <maintainerurl>https://www.pagantis.com/</maintainerurl>
      <targetplatform name="joomla" version="2.5"/>
   </update>
   <update>
-     <name>Pagamastarde</name>
+   <name>Paga+Tarde</name>
```

Comment:
PAgantis => Search in the whole code

Reply:
No, this Paga+Tarde texts are for the Fatsini Pagamastarde releases. This file is a roadmap, and points to the pagamastarde versions, we can delete the history in we want, but don't have sense to replace the words